src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,10 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard movie={movie} key={movie.ImdbId} />
```

Comment:
The property used as a key in the MovieCard component is incorrect. It should be 'movie.imdbId' instead of 'movie.ImdbId'. JavaScript is case-sensitive, and the property names from the original task description should be used as is.